execution/block-partitioner/src/v2/init.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright © Aptos Foundation
+
+use crate::{
+    get_anchor_shard_id,
+    v2::{
+        conflicting_txn_tracker::ConflictingTxnTracker, counters::MISC_TIMERS_SECONDS,
+        state::PartitionState, types::PreParedTxnIdx, PartitionerV2,
+    },
+};
+use rayon::{iter::ParallelIterator, prelude::IntoParallelIterator};
+use std::sync::RwLock;
+
+impl PartitionerV2 {
+    pub(crate) fn init(state: &mut PartitionState) {
```

Comment:
This function really belong in state.rs and also should be called from the constructor of the state?

Reply:
See [here](https://github.com/aptos-labs/aptos-core/pull/9488#discussion_r1294772987).